pyproject.toml

```diff
@@ -11,8 +11,8 @@ authors = [
     { name = "Pradyun Gedam", email = "pradyunsg@gmail.com" },
 ]
 requires-python = ">=3.9"
+license = "MIT"
```

Comment:
You could also add `license-files` key here.

Reply:
To 3.11, yes. I keep forgetting that!